### PR TITLE
update the readme for new features

### PR DESCRIPTION
**Pull request by justinwoo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,12 +128,26 @@ Then copy over `packages.json` into the directory:
...
+
 ### Update a package in the set
 
 Similar to the above, you will need to do the setup. You can then modify the version field to the version you wish to use. Once you have updated the package set, run the copy command and verify the package you have modified.
 
 Then you can make a pull request. Again, once Travis verifies your change, we will merge it into `master` and your change will be available in the next tag.
 
+### Add a package from Bower
+
+If you have Bower installed on your system, you can easily add a package from Bower to the package set. Follow the instructions above with the instructions written above. Then use the package name to add the package.
```
Comment (nwolverson) [inline README.md]: >  Follow the instructions above with the instructions written above

That's a little repetitive little repetitive

File: README.md [hunk 2]
```diff
@@ -79,8 +79,8 @@ A new package can be created using `psc-package init`. This will:
 
 To add a dependency, either:
 
-- Use the `install` command, which will update the project configuration automatically, or
-- Modify the `psc-package.json` file, and sync manually by running the `update` command.
+- Use the `install <package name>` command, which will update the project configuration automatically, or
+- Modify the `psc-package.json` file, and sync manually by running the `install` command.
```
Comment (nwolverson) [inline README.md]: Not sure if it's worth commenting "(previously `update`)" or that just confuses matters

File: README.md [hunk 3]
```diff
@@ -128,12 +128,26 @@ Then copy over `packages.json` into the directory:
...
 Similar to the above, you will need to do the setup. You can then modify the version field to the version you wish to use. Once you have updated the package set, run the copy command and verify the package you have modified.
 
 Then you can make a pull request. Again, once Travis verifies your change, we will merge it into `master` and your change will be available in the next tag.
 
+### Add a package from Bower
+
+If you have Bower installed on your system, you can easily add a package from Bower to the package set. Follow the instructions above with the instructions written above. Then use the package name to add the package.
+
+```
+psc-package install <purescript-something>
```
Comment (nwolverson) [inline README.md]: uh? Isn't this the wrong command?

**Review by nwolverson — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (justinwoo) [reply README.md]: new keyboard woes...

File: README.md (hunk 3 above)
Comment (justinwoo) [reply README.md]: brain fart

File: README.md [hunk 4]
```diff
@@ -128,12 +128,26 @@ Then copy over `packages.json` into the directory:
...
+
 ### Update a package in the set
 
 Similar to the above, you will need to do the setup. You can then modify the version field to the version you wish to use. Once you have updated the package set, run the copy command and verify the package you have modified.
 
 Then you can make a pull request. Again, once Travis verifies your change, we will merge it into `master` and your change will be available in the next tag.
 
+### Add a package from Bower
+
+If you have Bower installed on your system, you can easily add a package from Bower to the package set. Follow the instructions above. Then use the package name to add the package using the `add-from-bower` command:
```
Comment (nwolverson) [inline README.md]: Still not entirely sure what the instructions above are, aren't they mostly about things (updating, formatting, manually adding) that don't apply to adding from bower?

**Review by nwolverson — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (justinwoo) [reply README.md]: I guess, it just mirrors the previous on looking up to the top level explanation on how to work with package sets in general-ish. Could just remove the section altogether

**Pull request merged**